test/create_zarr_optimus.py

```diff
@@ -0,0 +1,298 @@
+import argparse
+import csv
+import re
+import gzip
+from scipy import sparse
+import numpy as np
+import zarr
+from zarr import Blosc
+
+""" key,  dataset name,   suffix of file,  metadata description  this is a list of groups to add to the zarr 
+    file, which can be  further extended
+"""
+
+ZARR_GROUP = {
+    'expression_matrix': ["expression", "cell_id", "gene_id",
+                          "gene_metadata_string", "gene_metadata_numeric", 
+                           "cell_metadata_string", "cell_metadata_numeric"
+                         ]
+}
+
+COMPRESSOR = Blosc(cname='lz4', clevel=5, shuffle=Blosc.SHUFFLE, blocksize=0)
+
+# the number of rows in a chunk for expression counts
+CHUNK_ROW_SIZE = 200
```

Comment:
> Pretty sure the spec says 10000.

Is there a programmatic representation of the `zarr` spec so we can test/validate this automatically rather than rely on correctly reading, interpreting and implementing the zarr spec?

Reply:
No not currently.